static/src/js/util/map/interactions/onClickShapefile.ts

```diff
@@ -85,24 +85,24 @@ const onClickShapefile = ({
   const updatedSelectedFeatures = [...currentSelectedFeatures]
 
   const currentQuery = {
-    boundingBox: spatialSearch.boundingBoxSearch,
-    circle: spatialSearch.circleSearch,
-    line: spatialSearch.lineSearch,
-    point: spatialSearch.pointSearch,
-    polygon: spatialSearch.polygonSearch
+    boundingBox: spatialSearch.boundingBoxSearch || [],
+    circle: spatialSearch.circleSearch || [],
+    line: spatialSearch.lineSearch || [],
+    point: spatialSearch.pointSearch || [],
+    polygon: spatialSearch.polygonSearch || []
   }
 
   const newQuery = getQueryFromShapefileFeature(feature)
   const [queryType] = Object.keys(newQuery) as SpatialQueryType[]
-  const [queryValue] = newQuery[queryType]
+  const queryValues = newQuery[queryType] || []
```

Comment:
Is there any way `newQuery[queryType]` can be undefined now that you have the defaults above?